content/v2.0/account-management/pricing-plans.md

```diff
@@ -69,3 +70,9 @@ To protect against any intentional or unintentional harm, Usage-based Plans incl
   - Unlimited PagerDuty, Slack, and HTTP notification endpoints
 
 _To request higher rate limits, contact [InfluxData Support](mailto:support@influxdata.com)._
+
+{{% cloud %}}
+#### VPC and other plans
```

Comment:
What are you referencing when you say "other plans?" You can probably update this to "VPC peering."

Reply:
Since it's a fairly unique plan, I thought it made sense visually set it off.